cabal.project

```diff
@@ -4,20 +4,3 @@ packages:
   serialise/
   binary-serialise-cbor/
   cborg-json/
-
-repository head.hackage
```

Comment:
`store-core` and `store` fail to build without patches from head.hackage: https://github.com/fpco/store/issues/144

Reply:
`store-core` and `store` now have compatible releases.